### PR TITLE
feat(animationCurveBlock): second round of ux fixes 

### PR DESCRIPTION
**Pull request by ragi96**
[Task](https://app.clickup.com/t/865c7g96v)

Shared:
- added getGutterSettings (used within Text-Block and Animation-Curve-Block)

### CODE REVIEW

File: packages/shared/src/settings/gutter.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import { SettingBlock, appendUnit, numericalOrPixelRule, presetCustomValue } from '@frontify/guideline-blocks-settings';
+import { GutterSpacing, gutterSpacingStyleMap } from './types';
+
+/**
+ * Returns background settings: background switch, background color
+ *
+ * @param options Options for the settings
+ * @param options.id Custom id for the gutters switch
+ * @returns {SettingBlock} Returns background settings
+ */
```
Comment (Walkeezy) [inline packages/shared/src/settings/gutter.ts:12]: Is this correct? Background settings?

Comment (ragi96) [reply packages/shared/src/settings/gutter.ts]: No copy pasta missed to adjust it

File: packages/animation-curve-block/src/AnimationCurveBlock.spec.ct.tsx [hunk 2]
```diff
@@ -152,20 +152,6 @@ describe('AnimationCurve Block', () => {
...
-                hasParameter: true,
-            },
-        });
-        mount(<AssetKitBlockWithStubs />);
-
-        cy.get(CARD_PARAMETERS_SELECTOR).should('css', 'cursor', 'copy');
-        cy.get(CARD_PARAMETERS_SELECTOR).click();
-        cy.get(CARD_PARAMETERS_SELECTOR).should('css', 'cursor', 'auto');
-    });
-
```
Comment (Walkeezy) [inline packages/animation-curve-block/src/AnimationCurveBlock.spec.ct.tsx:168]: Was this test removed intentionally?

**Review by Walkeezy — APPROVED** [review]
Works well!

File: packages/animation-curve-block/src/components/AnimationCanvas.tsx [hunk 3]
```diff
@@ -105,18 +105,27 @@ export const AnimationCanvas = ({
...
-            }
+            setAnimationFunction && setAnimationFunction(updatedAnimationFunction(animationValues, draggingPoint));
         },
         // eslint-disable-next-line react-hooks/exhaustive-deps
         [draggingPoint, viewBox.width, viewBox.height]
     );
 
+    const calculateXValue = (viewBoxX: number, viewBoxWidth: number) => {
+        const x = viewBoxX / viewBoxWidth;
+        if (x < 0) {
```
Comment (fulopdaniel) [inline packages/animation-curve-block/src/components/AnimationCanvas.tsx:120]: Optional, to consume less lines:
`return Math.max(Math.min(x, 1), 0)`

File: packages/animation-curve-block/src/components/AnimationCurveFlyout.tsx [hunk 4]
```diff
@@ -93,10 +93,9 @@ export const AnimationCurveFlyout = ({
 
     return (
         <Flyout
-            trigger={<></>}
+            trigger={(props, ref) => <div ref={ref as MutableRefObject<HTMLDivElement>} />}
```
Comment (fulopdaniel) [inline packages/animation-curve-block/src/components/AnimationCurveFlyout.tsx:96]: What is the purpose of this change?

Comment (ragi96) [reply packages/animation-curve-block/src/components/AnimationCurveFlyout.tsx:96]: Its to get rid of the trigger button which would be focusable but as its open via the toolbar this would be quite confusing for the user.

Or described by Martin:
`Accessibility: Edit animation button seems to be targeted twice when going through with tab. Second time there is no visual highlight, which could be confusing.`

File: packages/animation-curve-block/src/components/AnimationCanvas.tsx (hunk 3 above)
Comment (ragi96) [reply packages/animation-curve-block/src/components/AnimationCanvas.tsx:120]: Thanks. Adjusted and got rid of that function

**Review by fulopdaniel — APPROVED** [review]
